### ISSUE TITLE
Config key "source.wipe" may not be used

### ISSUE THREAD

**Issue by lds-lds** [canonical/microcloud#258]
Good morning

I set up an lxd cluster with microcloud this morning and  at the end of microcloud init i get this error: 

`Error: Config key "source.wipe" may not be used as member-specific key`

UPLINK is pending and no storage create

```
lxc network list
+--------+----------+---------+------+------+-------------+---------+---------+
|  NAME  |   TYPE   | MANAGED | IPV4 | IPV6 | DESCRIPTION | USED BY |  STATE  |
+--------+----------+---------+------+------+-------------+---------+---------+
| UPLINK | physical | YES     |      |      |             | 0       | PENDING |
+--------+----------+---------+------+------+-------------+---------+---------+
| ens2   | physical | NO      |      |      |             | 0       |         |
+--------+----------+---------+------+------+-------------+---------+---------+
| ens5   | physical | NO      |      |      |             | 0       |         |
+--------+----------+---------+------+------+-------------+---------+---------+
| vlan15 | vlan     | NO      |      |      |             | 0       |         |
+--------+----------+---------+------+------+-------------+---------+---------+
```

```
lxc init ubuntu:22.04
Creating the instance
Error: Failed instance creation: Failed creating instance record: Failed initialising instance: Failed getting root disk: No root device could be found
```
here are my snap configurations on ubuntu="22.04.3 LTS (Jammy Jellyfish)" on 3 scaleway instances 

snap install with:
```
for i in microceph microovn microcloud ; do snap install $i --cohort="+"; done && $ snap refresh lxd --channel=latest/stable --cohort="+"
```
```
# snap list
Name        Version                 Rev    Tracking       Publisher   Notes
core20      20230801                2015   latest/stable  canonical✓  base
core22      20231123                1033   latest/stable  canonical✓  base
lxd         5.0.2-838e1b2           24322  5.0/stable/…   canonical✓  -
microceph   0+git.4a608fc           793    quincy/stable  canonical✓  in-cohort
microcloud  1.1-04a1c49             734    latest/stable  canonical✓  in-cohort
microovn    22.03.3+snap1d18f95c73  349    22.03/stable   canonical✓  in-cohort
snapd       2.60.4                  20290  latest/stable  canonical✓  snapd
```

**Comment by lds-lds** [comment]
strange, work now !!!

**Comment by somewho** [comment]
got same error

reinstall does not help

**Comment by masnax** [comment]
If you're getting the `source.wipe` error, that means your LXD is out of date on some system. That does seem odd to me since you have `snap refresh lxd --channel latest/stable` in your first post. Can you confirm that each system is actually on the stable version (5.20)? You can check this with `lxc --version` on each system.

**Comment by somewho** [comment]
All 4 nodes are hardware identical and deployed Ubuntu 22.04 LTS by MaaS with same cloud init config:

#cloud-config
version: 1
packages:
  - openvswitch-switch
snap:
  commands:
    00: snap set system proxy.http="http://192.168.97.1:8000"
    01: snap set system proxy.https="http://192.168.97.1:8000"
    02: snap install lxd --cohort="+"
    03: snap install microceph --cohort="+"
    04: snap install microovn --cohort="+"
    05: snap install microcloud --cohort="+"
    06: snap refresh --cohort="+"

**Comment by masnax** [comment]
Hmm, so 22.04 LTS should come with LXD 5.0 preinstalled, so the `snap install lxd` line might not be grabbing the latest version. You may want to try `snap refresh --channel latest/stable`.

**Comment by tomponline** [comment]
You will need LXD 5.0.3 to support source.wipe option. Please use `snap refresh lxd` to get it.

**Comment by somewho** [comment]
>  `snap refresh --channel latest/stable`

That it! Perfect!

**Comment by masnax** [comment]
Fixed by  #261
